nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol

```diff
@@ -97,6 +109,12 @@ contract PolicyManager is Upgradeable {
         uint256 minRewardRate;
     }
 
+    struct Range {
+        uint256 min;
+        uint256 defaultValue;
```

Comment:
The presence of `defaultValue` within a 'range' will be less confusing once there are actual numerical values – because it will make sense that it falls in between the min and max.

Reply:
I'm not sure what you want to see, this lines - it's just a type, in different levels we have check for `default >= minimum & default <= maximum`. Could you elaborate?